hack/installer/entrypoint.sh

```diff
@@ -70,9 +70,14 @@ function create_iso() {
   mkdir -p /mnt/boot/isolinux
   cp -v /usr/local/src/syslinux/bios/core/isolinux.bin /mnt/boot/isolinux/isolinux.bin
   cp -v /usr/local/src/syslinux/bios/com32/elflink/ldlinux/ldlinux.c32 /mnt/boot/isolinux/ldlinux.c32
-  create_extlinux_conf /mnt/boot/isolinux/isolinux.conf
+  TALOS_USERDATA=default
+  TALOS_PLATFORM=iso
```

Comment:
Does this get translated into a kernel arg?

Reply:
It does. But in this case it only applies to the iso image. This is our way of telling Talos that it is currently in ISO mode.